spec/defines/vhost_spec.rb

```diff
@@ -2020,7 +2020,7 @@
             it { is_expected.to compile }
             it { is_expected.to contain_concat('25-rspec.example.com.conf') }
 
-            if (os_facts[:os]['family'] == 'RedHat' && os_facts[:os]['release']['major'].to_i > 6) ||
+            if (os_facts[:os]['family'] == 'RedHat' && os_facts[:os]['release']['major'].to_i >= 7) ||
                (os_facts[:os]['name'] == 'SLES' && os_facts[:os]['release']['major'].to_i > 11)
```

Comment:
This can be simplified to:
```suggestion
            if os_facts[:os]['family'] == 'RedHat' || os_facts[:os]['name'] == 'SLES'
```

But really, I think the whole `if` condition can be dropped. It's true for everything now.